src/dashboard/src/components/api/views.py

```diff
@@ -582,6 +565,8 @@ def path_metadata(request):
         return helpers.json_response(body, status_code=201)
 
 
+# TODO should this have auth?
+@_api_endpoint(expected_methods=['GET'])
 def processing_configuration(request, name):
```

Comment:
This had no auth or verb restriction previously - needs to be tested to see if it still works with external consumers.

This returns an XML document as the body instead of JSON, which is different from the rest of the API. Should this be changed?